### ISSUE TITLE
Initial creation no longer tracked

### ISSUE THREAD

**Issue by russellquinn** [adamcooper/vestal_versions#4]
Vestal Versions stopped tracking the initial creation of my records sometime around the 2010-09-20 commits.

**Comment by adamcooper** [comment]
I'm not seeing this behaviour.  I created a new rails app and was not able to reproduce this issue.

Could you give me some more information about it?  Are any versions being created?  Are you passing ':initial_version => true' to the versioned method?  What does your versions table look like?

I do know that was a schema change required on the dates you mentioned...  An integer reverted_from column was added.  That may be the culprit.

Thanks Adam.

**Comment by russellquinn** [comment]
Hey Adam,

OK, I've tracked this down. My code during record creation is:

```
@article = Article.new params[:article]
@article.updated_by = "Created by #{@session_user.name}"
@article.merge_version do
    @save_result = @article.save
end
```

The Article model has some complex relationships and the version of vestal_versions previous to 2010-09-20 generated a version 2 for this code. i.e. It was never making an initial version, but was in fact making a version 2 for this.

My view code was then subtracting 1 from the version number it displayed to compensate for this.

At some point, I guess vestal_versions fixed the problem and it no longer created a version 2 for the above code. I forgot that I was compensating for the version number in the view and figured that it was no longer making a version 1.

I now set ':initial_version => true' and am getting my version 1 saved to the versions table. I have also removed my version-number-compensation code from my views and everything is good, with the side effect of all old records now starting from version 2, but that's no problem.

So, basically, I think my app was displaying desired behaviour because of a previous vestal_versions bug and my compensation code. It now all _really_ works correctly :)
